### PR TITLE
Feature/976 apostrophes vs capitals in snippets

### PR DESCRIPTION
**Pull request by WillGibson**
Addresses "Apostrophes causing unexpected capitals in snippets #976" with another PR on Behat/Transliterator (https://github.com/Behat/Transliterator/pull/18).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ cache:
 branches:
   only:
     - master
+    # ToDo: Remove this line once all done
+    - feature/976-apostrophes-vs-capitals-in-snippets
```
Comment (Taluu) [inline .travis.yml]: you actually don't need that line, PRs are automatically tested

Comment (WillGibson) [inline .travis.yml]: Ah, yes, it wasn't a PR before. I'll remove that.

File: composer.json [hunk 2]
```diff
@@ -13,11 +13,18 @@
...
+            "type": "git",
+            "url": "https://github.com/BiffBangPow/Transliterator"
+        }
+    ],
+
     "require": {
         "php": ">=5.3.3",
         "ext-mbstring": "*",
         "behat/gherkin": "^4.4.4",
-        "behat/transliterator": "~1.0",
```
Comment (WillGibson) [inline composer.json]: This will need sorting out before merging.

**Comment by Taluu** [comment]
Actually I am unsure of the interest of this PR, as it concerns only the pr made in the translit (this shouldn't be tested here, it works without it)

**Comment by WillGibson** [comment]
@Taluu I think there would need to be a PR here because I added a scenario into features/snippets.feature because it is already testing the area of snippet generation in here, so it seemed write to test the new behaviour in the same way.

**Comment by Taluu** [comment]
The generation is already tested actually, so it doesn't make sense to test every single variant on the main repo IMO. If it is tested in the translit repo, this should be enough

**Comment by WillGibson** [comment]
What do you think here @stof? Do you want these tests here? Or do you agree with @Taluu?

File: composer.json (hunk 2 above)
Comment (everzet) [inline composer.json]: @WillGibson are you ok to bump it?

Comment (WillGibson) [inline composer.json]: Yup, give me a few minutes...

Comment (WillGibson) [inline composer.json]: @everzet That's done now, and I squashed it all into a single commit. Just waiting on the build.

**Pull request merged**